src/iris/api.py

```diff
@@ -1685,6 +1694,13 @@ def on_post(self, req, resp):
             if not app_template_count:
                 raise HTTPBadRequest('No plan template actions exist for this app')
 
+        incident_data = {
```

Comment:
Are you sure you don't want to include incident_id as part of this data so that people can link them on iris as well?

Reply:
good catch, added that